content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -57,58 +60,65 @@ For example:
 
 1. List all of the pods in a given namespace.
 
-        GET /api/v1/namespaces/test/pods
-        ---
-        200 OK
-        Content-Type: application/json
-        {
-          "kind": "PodList",
-          "apiVersion": "v1",
-          "metadata": {"resourceVersion":"10245"},
-          "items": [...]
-        }
+   ```console
+   GET /api/v1/namespaces/test/pods
+   ---
+   200 OK
+   Content-Type: application/json
```

Comment:
mini nit: HTTP/1.1 usually has a blank line between headers and body; I'd include that here (it looks odd to me without it)